src/system/applications/dialogs/attack-configuration.ts

```diff
@@ -216,7 +222,7 @@ export class AttackConfigurationDialog extends ComponentHandlebarsApplicationMix
         this.data.skillTest.data.mod = rank + value;
         this.data.skillTest.plotDie = plotDie;
 
-        this.data.defaultAttribute = !isNone(attribute) ? attribute : undefined;
+        this.data.defaultAttribute = !isNull(attribute) ? attribute : undefined;
```

Comment:
Same question here, but instead we can also use the nullish coalescing operator for this.